login/src/Components/register.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import '../App.css';
+import Axios from 'axios';
```

Comment:
Don't know if this really matters, but this could be lowercase axios. Capitalization is usually used for Components in React and classes in javascript